ElectronNET.CLI/Commands/InitCommand.cs

```diff
@@ -147,10 +148,19 @@ private static bool EditCsProj(string projectFile)
                 stream.SetLength(0);
                 stream.Position = 0;
 
-                xmlDocument.Save(stream);
+                var xws = new XmlWriterSettings
+                {
+                    OmitXmlDeclaration = true,
+                    Indent = true
+                };
+                using (XmlWriter xw = XmlWriter.Create(stream, xws))
+                {
+                    xmlDocument.Save(xw);
+                }
 
-                Console.WriteLine($"{ConfigName} added in csproj!");
             }
+
+            Console.WriteLine($"{ConfigName} added in csproj!");
```

Comment:
Mh - why did you move the Console.WriteLine out of the if(...) section? In theory you can run "init" any time, but we only will add our file once - thats why it is inside the if statement

Reply:
i wanted to move it out for the top level using block in fact i might have done that too fast (no more computer will update tomorrow)

Reply:
it was not in any if() section right ? just the last line of the using()
now it is not

Reply:
AHHH  - whoops - sorry! You are completely right! Looks good!